apps/web/.env.example

```diff
@@ -1,9 +1,21 @@
+## Auth with NextAuth - https://nextauth.org/
 NEXTAUTH_SECRET= # Linux: `openssl rand -hex 32` or go to https://generate-secret.now.sh/32
+
+## Twitter - https://developer.twitter.com/en/apps
 TWITTER_BEARER_TOKEN=
 TWITTER_CLIENT_ID=
 TWITTER_CLIENT_SECRET=
 TWITTER_PROFILE_ID=#2313873457
 TWITTER_PROFILE_USER=#sseraphini
+
+## Discord and Discord CronJob - display notifications on discord channel #tweets
 WEBHOOK_DISCORD= #get this url creating a channel then create a webhook, copy the url and paste here
 CRONJOB_HEADER_KEY=# add a key here, should be the same as in cronjob.org, sample: x-servicename-supersecret-key
 CRONJOB_HEADER_VALUE=# add a value here, should be same as in cronjob.org , sample: 1q2w3e4r5t6y7u8i9o0p
+
+## Sentry - logging and debugging in production
+SENTRY_DSN=
+NEXT_PUBLIC_SENTRY_DSN=
+SENTRY_ORG=

```

Comment:
are we using this?

Reply:
only sentry_dsn, removing the others here